src/django/api/middleware.py

```diff
@@ -13,24 +12,18 @@
 
 from api.models import RequestLog
 from api.limits import get_api_block
+from oar.rollbar import report_error_to_rollbar
 
-
-def _report_error_to_rollbar(request, auth):
-    ROLLBAR = getattr(settings, 'ROLLBAR', {})
-    if ROLLBAR:
-        import rollbar
-        rollbar.report_exc_info(
-            sys.exc_info(),
-            extra_data={'auth': auth})
+logger = logging.getLogger(__name__)
```

Comment:
Why to we need logger here?

Reply:
That's my leftover. I will remove it, thank you!